src/main/java/ch/njol/skript/effects/EffMessage.java

```diff
@@ -37,20 +40,34 @@
 import ch.njol.skript.util.chat.BungeeConverter;
 import ch.njol.skript.util.chat.ChatMessages;
 import ch.njol.util.Kleenean;
+import net.md_5.bungee.api.chat.BaseComponent;
+import net.md_5.bungee.api.chat.TextComponent;
 
 @Name("Message")
 @Description({"Sends a message to the given player. Only styles written",
-		"in given string or in <a href=expressions.html#ExprColoured>formatted expressions</a> will be parsed."})
+		"in given string or in <a href=expressions.html#ExprColoured>formatted expressions</a> will be parsed.",
+		"Adding an optional sender allows the messages to be sent as if a specific player sent them.",
+		"This is useful with Minecraft 1.16.4's new chat ignore system."})
 @Examples({"message \"A wild %player% appeared!\"",
 		"message \"This message is a distraction. Mwahaha!\"",
 		"send \"Your kill streak is %{kill streak::%uuid of player%}%.\" to player",
 		"if the targeted entity exists:",
-		"	message \"You're currently looking at a %type of the targeted entity%!\""})
-@Since("1.0, 2.2-dev26 (advanced features)")
+		"\tmessage \"You're currently looking at a %type of the targeted entity%!\"",
+		"on chat:",
+		"\tcancel event",
+		"\tsend \"[%player%] >> %message%\" to all players from player"})
+@RequiredPlugins("Minecraft 1.16.4+ for optional sender")
+@Since("1.0, 2.2-dev26 (advanced features), INSERT VERSION (optional sender)")
 public class EffMessage extends Effect {
 	
+	private static final boolean SUPPORTS_SENDER;
+	
 	static {
-		Skript.registerEffect(EffMessage.class, "(message|send [message[s]]) %strings% [to %commandsenders%]");
+		SUPPORTS_SENDER = Skript.classExists("org.bukkit.command.CommandSender$Spigot") && Skript.methodExists(CommandSender.Spigot.class, "sendMessage", UUID.class, BaseComponent.class);
```

Comment:
I don't know if this really matters, but would it be better to just initialize the variable above?

Reply:
Yeah I could do that.

Reply:
done 👍